### PR TITLE
KFSPTS-20244 upgrade to the 11-12-2020 version of financials

### PR DESCRIPTION
**Pull request by jhulslander**
Our existing DocumentReindexStep takes in a list of document numbers to index where the new kualiCo patch job re indexes based on document type.  I think we need to keep the existing batch job, but will follow up with Cathy.  I did update DocumentReindexStep to use the save index base code service as the new batch job so both jobs index documents in the same fashion

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/sys/service/impl/CuGeneralLedgerPendingEntryServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+import edu.cornell.kfs.sys.service.CuGeneralLedgerPendingEntryService;
+
+public class CuGeneralLedgerPendingEntryServiceImpl extends GeneralLedgerPendingEntryServiceImpl
+        implements CuGeneralLedgerPendingEntryService {
+    private static final Logger LOG = LogManager.getLogger(CuGeneralLedgerPendingEntryServiceImpl.class);
+    protected HomeOriginationService homeOriginationService;
+
+    /*
+     * With the 11/20/2020 patch, this function was removed from GeneralLedgerPendingEntryServiceImpl,
```
Comment (db483) [inline src/main/java/edu/cornell/kfs/sys/service/impl/CuGeneralLedgerPendingEntryServiceImpl.java:26]: should this version be 11/12/2020?

**Review by db483 — CHANGES_REQUESTED** [review]
Changes look good. There is one comment that should be updated to the correct version of financials. I think it would be good if the PR title was also updated to use the full version date 11-12-20202. Thank you

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
I've updated the comment as suggested

File: src/main/java/edu/cornell/kfs/sys/service/impl/CuGeneralLedgerPendingEntryServiceImpl.java (hunk 1 above)
Comment (chadbower) [reply src/main/java/edu/cornell/kfs/sys/service/impl/CuGeneralLedgerPendingEntryServiceImpl.java]: please retest this

Comment (chadbower) [reply src/main/java/edu/cornell/kfs/sys/service/impl/CuGeneralLedgerPendingEntryServiceImpl.java]: test this please

**Comment by chadbower** [comment]
test this please

File: src/main/java/edu/cornell/kfs/sys/service/impl/CuGeneralLedgerPendingEntryServiceImpl.java (hunk 1 above)
Comment (chadbower) [reply src/main/java/edu/cornell/kfs/sys/service/impl/CuGeneralLedgerPendingEntryServiceImpl.java]: retest this please

**Comment by jhulslander** [comment]
retest this please

**Comment by chadbower** [comment]
retest this please

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
this is ready for review again, but the details URL isn't working correct yet

**Review by db483 — APPROVED** [review]
Changes look good.

**Comment by chadbower** [comment]
retest this please

**Pull request merged**